engines/account/app/controllers/account/api/v1/login_controller.rb

```diff
@@ -16,7 +17,7 @@ def login
         private
 
         def login_params
-          params.permit(:email, :password)
+          params.permit(:username, :email, :password)
```

Comment:
This is the typical case when an API should be really clear. Even the current implementation is not efficient.
I'll make a proposal to explain how I would do it.